controllers/v1/userExtension.js

```diff
@@ -372,9 +492,9 @@ module.exports = class userExtension extends Abstract {
     */
 
   /**
-   * create User
+   * Bulk user sample csv 
    * @method
-   * @name create
+   * @name bulkUserSampleCsvDwonload
```

Comment:
@rakeshSgr - Rename this to bulkUploadSampleFile

Reply:
resolved